connector/src/main/scala/com/microsoft/kusto/spark/utils/KustoDataSourceUtils.scala

```diff
@@ -113,6 +113,8 @@ object KustoDataSourceUtils {
     val applicationKey = parameters.getOrElse(KustoSourceOptions.KUSTO_AAD_APP_SECRET, "")
     val applicationCertPath = parameters.getOrElse(KustoSourceOptions.KUSTO_AAD_APP_CERTIFICATE_PATH, "")
     val applicationCertPassword = parameters.getOrElse(KustoSourceOptions.KUSTO_AAD_APP_CERTIFICATE_PASSWORD, "")
+    val tokenProviderCoordinates = parameters.getOrElse(KustoSourceOptions.KUSTO_TOKEN_PROVIDER, "")
```

Comment:
Coordinates? did you mean callback?

```suggestion
    val tokenProviderCallback = parameters.getOrElse(KustoSourceOptions.KUSTO_TOKEN_PROVIDER, "")
```

Reply:
i meant the class path to the tokenProvider
i.e 
com.microsoft.kusto.spark.authentication.TokenProvider